connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -343,10 +343,10 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
             extracted.length > 0 &&
             extracted.length <= MAX_SMALL_DOCUMENT_TXT_LEN
           ) {
-            const validatedUrl = validateUrl(url);
+            const validatedUrl = validateUrl(request.url);
```

Comment:
Wait maybe we also want to rename the url var to rootUrl? Pit of success for next engineers